Addons/NetworkZones/NetworkZones_MySql.cs

```diff
@@ -0,0 +1,107 @@
+
+using UnityEngine;
+using UnityEngine.Networking;
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using System.Data;
+using MySql.Data;								// From MySql.Data.dll in Plugins folder
+using MySql.Data.MySqlClient;                   // From MySql.Data.dll in Plugins folder
+
+
+using SqlParameter = MySql.Data.MySqlClient.MySqlParameter;
+
+
+/// <summary>
+/// Database class for mysql
+/// Port of the sqlite database class from ummorpg
+/// </summary>
+public partial class Database
+{
+
+    static void Initialize_Zone()
+    {
+        ExecuteNonQueryMySql(@"
+        CREATE TABLE IF NOT EXISTS character_scene (
+            `character` VARCHAR(16) NOT NULL, 
```

Comment:
add a foreign key to characters with this field with on delete cascade.   That way if the character is deleted,  this table is automatically cleaned up as well.

Reply:
FOREIGN KEY(`character`) ON DELETE CASCADE ON UPDATE CASCADE,
Maybe name can changed also in future, GM or something else